runner/src/main.rs

```diff
@@ -517,6 +517,26 @@ fn run_ci() -> Step {
                     server_rust_target: None,
                 },
             }),
+            // Package the Veracruz Demo application in a Docker image.
+            run_examples(&RunExamples {
```

Comment:
This test was only needed for the `hello_world` example, just to check that creating a Docker image works.
Not sure if it's necessary for the `veracruz_demo` example, since it's already being automatically run as part of `run-examples`